packages/web-client/app/helpers/format-token-amount.ts

```diff
@@ -6,6 +6,44 @@ import { fromWei } from 'web3-utils';
 
 type FormatTokenAmountHelperParams = [BN, number];
 
+export function formatTokenAmount(
```

Comment:
we have a huge amount of currency utils for formatting tokens and fiat currencies in the SDK (currency-utils) that allow us to configure significant digits for different currencies, etc. that we use for the wallet app. is there a reason we don't use that in the web client? it would be great for number formatting to be consistent across these clients.

Reply:
Captured here: https://linear.app/cardstack/issue/CS-1605/make-web-clients-format-token-amount-helper-delegate-to-the-sdks